### ISSUE TITLE
Size of the DSDT file is too large (> 64k). Try to build the template from another computer

### ISSUE THREAD

**Issue by lzkcmy** [nsmfoo/antivmdetection#37]
What shall I do?What's the problem?

**Comment by eyesonly** [comment]
I too get this problem, and have tried running the script on all of my machines. I assume this then results in the error "ACPI tables bigger than 64KB (VERR_TOO_MUCH_DATA) when attempting to start the guest.

Not sure what I can do - perhaps download a DSDT file found from the internet?

**Comment by lzkcmy** [comment]
I have solved  this problem, you need to run the code on an old computer! (The shell script of the virtualbox need a DSDT file which sholud be  smaller than 64KB.)

**Comment by nsmfoo** [comment]
That is correct, is a know issue with Virtualbox, there is requests to change the limit as it makes most new machines unusable. So as @lzkcmy wrote, boot up and old computer and run the script on that one, later on you can use your new computer

**Comment by ryanvillarreal** [comment]
What is meant by an "old" computer?  x86?

**Comment by dashjuvi** [comment]
What if you dont have an "old computer"? Cant something be done to fix that?

**Comment by SalehAce1** [comment]
Will this ever get fixed? My computer is 4 years old and it still gives me that error.

**Comment by elanfer** [comment]
I got the same problem, now I found an old HP notebook to execute the script. You can use my DSDT files until the problem is solved (maybe more a Virtualbox related issue...)

hp_pavilion_dsdt.tar.xz
https://myshare.uni-osnabrueck.de/f/0af92f7d3e88402b843f/
sha1: b47bdcefde76736204431ac0a0f9aa7d52366810

**Comment by MasterCATZ** [comment]
facepalm none of my old bootable machines from last decade will so this :S 
unless I did up my old p4 or p2 

https://www.tonymacx86.com/dsdt-database
or just get some random one from here

**Comment by nsmfoo** [comment]
@MasterCATZ thanks for the URL, could be very useful for people not being able to create their own. Also buggin Oracle about the issue might be a way forwards as well.. maybe one of these days they will make a change ...

**Comment by b-kitt** [comment]
How do I substitute a suitable dsdt for the one used by the script/ provided by an unsuitable computer?

**Comment by looi-wh** [comment]
To anybody still figuring it out:
download a random one from the link below:
https://www.tonymacx86.com/dsdt-database

rename it from *.aml to *.bin where * is your current .bin file.
run the shell script after that, it should work

**Comment by nsmfoo** [comment]
@looi-wh good comment! If you like, you are more than welcome to create a PR for an updated to the README file. If not I will add this shortly.

/Mikael

**Comment by looi-wh** [comment]
@nsmfoo but the DSDT file from other sources didnt worked for me. tried everything. any chance to have someone send over theirs?

**Comment by Phasip** [comment]
Just a small comment on this issue.
I attempted using the ```DSDT-GA-EP35C-DSR3-2.1-F4.aml``` from https://www.tonymacx86.com/dsdt-database (Socket 775, P35) which lets virtualbox start but the windows installer stops at ACPI Error sadscreen (the modern bluescreen)
Removing the customtable line from virtualbox allows windows installer to continue.
If someone has one that is known to work, that would be great

**Comment by nsmfoo** [comment]
As this is indeed an issue, I will try to poke the VBox developers to see if they would consider bumping the size limitation (again), which is a very artificial limit to begin with.

**Comment by nsmfoo** [comment]
Added a new topic in the VBOX forum. If you feel that solving this issue, I recommend that you head over to the VBOX forum and post a comment

**Comment by N3X15** [comment]
Please link the thread here so we don't have to hunt for it.

**Comment by nsmfoo** [comment]
Yes, that should have been done previously sorry and thanks for the suggestion!

https://forums.virtualbox.org/viewtopic.php?f=9&t=102733&hilit=nsmfoo
https://www.virtualbox.org/ticket/15043

I did plan to try to compile a version with the size changed, to see what the result was and then possible create a binary patch. Maybe that would be a good way to spend the holidays, let's see =)

**Comment by QiangF** [comment]
This one seems to work.
https://github.com/cardoe/qemu/blob/master/pc-bios/q35-acpi-dsdt.aml

**Comment by LukasJerabek** [comment]
@QiangF worked for me, thanks

**Comment by LukasJerabek** [comment]
oh too fast, I have acpi bios error later on booting...

**Comment by LukasJerabek** [comment]
Actually, it does work, I just needed to make a whole fresh setup, not trying to make work already failed one...
